app/web/features/profile/view/leaveReference/formSteps/submit/SubmitReference.tsx

```diff
@@ -140,8 +144,18 @@ export default function SubmitReference({
     }
   };
 
-  const redirectToHome = () => {
-    router.push(`${baseRoute}`);
+  const redirectToThankYouPage = () => {
+    if (
+      referenceType === referenceTypeRoute[ReferenceType.REFERENCE_TYPE_FRIEND]
+    ) {
+      router.push(
+        `${leaveReferenceBaseRoute}/${referenceType}/${userId}/${referenceStepStrings[4]}`,
```

Comment:
Should these be in `routes.ts`?

Reply:
It's using the `leaveReferenceBaseRoute` but otherwise it's a bit tricky since the url is dynamic and needs data from the component.

Reply:
Ah, no worries